hack/manifest-gen/manifest-gen.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+
+# Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+# or more contributor license agreements. Licensed under the Elastic License;
+# you may not use this file except in compliance with the Elastic License.
+
+# Script to generate an ECK installation manifest
+
+set -euo pipefail
+
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+CHART_DIR="${SCRIPT_DIR}/assets/charts/eck"
+
+update_chart() {
+    local ALL_CRDS="${SCRIPT_DIR}/../../config/crds/all-crds.yaml"
+    local VERSION
+    VERSION=$(cat "${SCRIPT_DIR}/../../VERSION")
+
+    cp -f "$ALL_CRDS" "${CHART_DIR}/crds/all-crds.yaml"
+    sed -i -E "s#version: [0-9]+\.[0-9]+\.[0-9]+#version: $VERSION#" "${CHART_DIR}/values.yaml"
+    sed -i -E "s#appVersion: [0-9]+\.[0-9]+\.[0-9]+#appVersion: $VERSION#" "${CHART_DIR}/Chart.yaml"
```

Comment:
This ends up with the below on MacOSX.
```
$ ls -a assets/charts/eck/Chart*
assets/charts/eck/Chart.yaml    assets/charts/eck/Chart.yaml-E
$ ls assets/charts/eck/values*
assets/charts/eck/values.yaml   assets/charts/eck/values.yaml-E
```

Reply:
Interesting... looks like `-i` is consuming `-E` as the backup suffix. It shouldn't do that though. What is the version of sed you have on your machine?

Reply:
Can't check the version unfortunately. Apparently it's [FreeBSD sed from 2005](https://unix.stackexchange.com/a/131940). For [.ci/Makefile](https://github.com/elastic/cloud-on-k8s/blob/b5dc5166aa35a079dd01b7ba9a0954589cc17605/.ci/Makefile#L27) we've ended up with `sed ... > tmp && mv tmp ...`.

Reply:
As far as I know in place edits that work on both gnu sed and freebsd aren't possible without something like that Makefile david linked